components/plan-build-ps1/habitat/plan.ps1

```diff
@@ -44,12 +43,6 @@ function Invoke-Build {
 
 function Invoke-Install {
     New-Item "$pkg_prefix\bin" -ItemType Directory -Force | Out-Null
+    Copy-Item "$PLAN_CONTEXT\..\bin\*" "$pkg_prefix\bin" -Force
     Copy-Item "$bin" "$pkg_prefix\bin\$bin" -Force
 }
-
-# Turn the remaining default phases into no-ops
-function Invoke-Download {}
-
-function Invoke-Verify {}
-
-function Invoke-Unpack {}
```

Comment:
we use to have to provide empty functions and a dummy source file. We don't need to do that anymore.